service/src/main/kotlin/fi/espoo/evaka/EspooConfig.kt

```diff
@@ -1,17 +1,15 @@
-// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```

Comment:
Was dropping the copyright text out intentional?

Reply:
No :grimacing: